pkg/transformers/namespace.go

```diff
@@ -97,5 +97,32 @@ func (o *namespaceTransformer) Transform(m resmap.ResMap) error {
 		}
 
 	}
+	o.updateClusterRoleBinding(m)
 	return nil
 }
+
+func (o *namespaceTransformer) updateClusterRoleBinding(m resmap.ResMap) {
+	for id := range m {
+		if id.Gvk().Kind == "ClusterRoleBinding" {
+			objMap := m[id].UnstructuredContent()
+			subjects := objMap["subjects"].([]interface{})
+			for i := range subjects {
+				subject := subjects[i].(map[string]interface{})
+				kind, foundk := subject["kind"]
+				name, foundn := subject["name"]
+				if foundk && foundn {
+					for objId := range m {
+						if objId.Gvk().Kind == kind.(string) && objId.Name() == name.(string) {
```

Comment:
Any reason we wouldn't use `selectByGVK` here?

Reply:
We should probably add a second conditional to `continue` if GVK is in `skipNamespacePathConfigs`.

Reply:
`selectByGVK` doesn't include the object name. Here we want find the exact object referred by subject. So we need `GVK` and `name`.